recipe/patches/001_relax_reqs.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/settings.ini b/settings.ini
+index d5e5157..66e4e06 100644
+--- a/settings.ini
++++ b/settings.ini
+@@ -15,7 +15,7 @@ language = English
+ custom_sidebar = True
+ license = apache2
+ status = 2
+-requirements = numpy>=1.21.6 pandas>=1.3.5 torch>=1.12.1 pytorch-lightning==1.6.5 ray[tune]==2.0.1
++requirements = numpy>=1.21.6 pandas>=1.3.5 torch>=1.12.1 pytorch-lightning==1.9.3 ray[tune]==2.3.0
```

Comment:
Does this help with `pip check`? We are doing `--no-deps` and `--no-build-isolation` anyway, so `pip` won't try to download these during the build.

Reply:
Yeah, it simply allows us to still call `pip check`